easybuild/framework/easyblock.py

```diff
@@ -2126,14 +2148,16 @@ def _sanity_check_step(self, custom_paths=None, custom_commands=None, extension=
         # run sanity check commands
         for command in commands:
 
-            out, ec = run_cmd(command, simple=False, log_ok=False, log_all=False)
+            out, ec = run_cmd(command, simple=False, log_ok=False, log_all=False, trace=False)
             if ec != 0:
                 fail_msg = "sanity check command %s exited with code %s (output: %s)" % (command, ec, out)
                 self.sanity_check_fail_msgs.append(fail_msg)
                 self.log.warning("Sanity check: %s" % self.sanity_check_fail_msgs[-1])
             else:
                 self.log.info("sanity check command %s ran successfully! (output: %s)" % (command, out))
 
+            trace_msg("running command '%s'... %s" % (command, ('FAILED', 'OK')[ec == 0]))
```

Comment:
I wouldn't to this here. Put such a thing inside `run_cmd`.

Reply:
I'm deliberately not using the standard trace message printed for `run_cmd` here because this is the sanity check; the output of the command doesn't matter here, just whether it exits with `0` (`OK`) or not (`FAILED`)